apis/r/README.md

```diff
@@ -29,10 +29,11 @@ $ install.packages('tiledbsoma', repos = c('https://tiledb-inc.r-universe.dev',
 
 * Clone this repository: `git clone https://github.com/single-cell-data/TileDB-SOMA.git`
 * Change into the R API package directory: `cd TileDB-SOMA/apis/r`
-* Optionally, clean the file in to the repo: `./cleanup` (this is not needed the first time)
+* Optionally, clean the files in the repo: `./cleanup` (this is not needed the first time)
 * Optionally, update the `libtiledbsoma` sources: `./copy_source.sh` (which updates the includes tarball of `libtiledbsoma`).
-* Build the R package source tarball from the repository sources: `R CMD build .` (which will also build `libtiledbsoma` from source; other dependencies are required as described in the previous section).
-* Check and test the package from the tarball: `R CMD check tiledbsoma_*.tar.gz`
+* Build the R package source tarball from the repository sources: `R CMD build .` (which will also build `libtiledbsoma` from source; other dependencies are required as described in the previous section)
+* Check and test the package from the tarball: `R CMD check --no-vignettes --no-manual tiledbsoma_*.tar.gz`
+  * For quicker iteration, after `Rscript -e 'install.packages("pkgbuild")'`, run `Rscript -e 'testthat::test_local(".")'`
```

Comment:
I suspect we can skip `install.packages("pkgbuild")` here. It is no suggested by `testthat` (whereas `pkgload` is already dependency -- and used in the code.  Typo on your end?

Also, I think from the package directory we need `testthat::test_local("tests/testthat")`

Reply:
oops yes of course thank you!!